### PR TITLE
Update .editorconfig

### PR DESCRIPTION
**Pull request by evpobr-collaborator**

### CODE REVIEW

**Review by evpobr — CHANGES_REQUESTED** [review]
Can you also add .cpp extension?

File: .editorconfig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 root = true
 
-[*.{h,c]
+[*.{h,c}]
```
Comment (evpobr) [inline .editorconfig]: # Add .cpp extension

```suggestion
[*.{h,c,cpp}]
```

Comment (evpobr-collaborator) [inline .editorconfig]:
```suggestion
[*.{h,c,cpp}]
```

**Review by evpobr — APPROVED** [review]

**Review by evpobr — APPROVED** [review]

**Review by evpobr — CHANGES_REQUESTED** [review]
Can you add markdown extension?

File: .editorconfig [hunk 2]
```diff
@@ -15,3 +15,6 @@ indent_size = 2
 [*.yml]
 indent_style = space
 indent_size = 2
+
+[*.{md}]
```
Comment (evpobr-collaborator) [inline .editorconfig]:
```suggestion
[*.{md,markdown}]
```

**Review by evpobr — APPROVED** [review]
This looks OK.

**Pull request merged**